opencga-app/app/scripts/azure/README.md

```diff
@@ -0,0 +1,164 @@
+- Search for **Data Lake Storage Gen1** (Gen2 might be available in the future)
+- Make sure to deploy it in the same region as your OpenCGA VM
+
+### Azure HDInsights - HBase (optional)
+If you want to leverage OpenCGA's more performant Hadoop backend instead of MongoDB, you have to create an Azure HDInsight instance with HBase.
+
+- Create a new resource
+- Search for **HDInsight**
+- Step 1: Click on the Custom tab (instead of Quick Create)
+- Step 1: Select **HBase 1.1.2 (HDI 3.6)** as the Cluster type
+- Step 1: Deploy it in the same region as your OpenCGA VM
+- Step 2: Place it in the same Virtual Network as your OpenCGA VM (select the virtual network that the VM has created when it was deployed)
+- Step 3: Configure Storage with either _Azure Storage_ or (optionally) the newly created _Data Lake Storage Gen1_ instance
+- Step 5: Depending on the required performance any cluster size is possible from 2 region nodes with minimum specs to more nodes with higher specs
+
+
+## Setup resources
+After successfully deploying the resources, they have to be  configured
+
+### OpenCGA Virtual Machine
+SSH into your machine (replace username and IP address)
+
+```
+ssh azure@139.349.12.49
+````
+
+Install the latest updates
+```
+sudo apt-get update
+sudo apt-get upgrade -y
+sudo apt-get install software-properties-common -y
+```
+
+Install Java SDK
+
+```
+sudo -- sh -c 'echo oracle-java8-installer shared/accepted-oracle-license-v1-1 select true | debconf-set-selections &&  add-apt-repository -y ppa:webupd8team/java &&  apt-get update &&  apt-get install -y oracle-java8-installer &&  rm -rf /var/lib/apt/lists/* &&  rm -rf /var/cache/oracle-jdk8-installer'
+```
+
+Install Redis, Tomcat, Maven and Git
+
+```
+sudo add-apt-repository universe && sudo apt-get update && sudo apt-get install redis-server redis-tools tomcat8 maven git-core -y
+```
+
+Install MongoDB 3.6
+
+```
+sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 2930ADAE8CAF5059EE73BB4B58712A2291FA4AD5
+echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu xenial/mongodb-org/3.6 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-3.6.list
+sudo apt-get update
+sudo apt-get install -y mongodb-org
+sudo systemctl enable mongod.service # start on boot
+```
+
+Install Solr
+
+```
+wget https://www-eu.apache.org/dist/lucene/solr/7.5.0/solr-7.5.0.tgz
+tar xvfz solr-7.5.0.tgz
+sudo ./solr-7.5.0/bin/install_solr_service.sh ./solr-7.5.0.tgz
+```
+
+Clone OpenCGA
+```
+git clone -b master https://github.com/opencb/opencga.git
+cd opencga/
+```
+
+**Option 1: Build OpenCGA with MongoDB**
+```
+mvn clean install -DskipTests
+```
+
+**Option 2: Build OpenCGA with Hadoop**
+```
+mvn clean install -DskipTests -Dstorage-hadoop -Popencga-storage-hadoop-deps -Phdp-2.6.0 -DOPENCGA.STORAGE.DEFAULT_ENGINE=hadoop
+```
+
+Move OpenCGA to `/opt/opencga`
+```
+sudo mkdir /opt/opencga
+sudo cp -r ~/opencga/build/* /opt/opencga
+```
+
+Copy OpenCGA build to Tomcat server
```

Comment:
This is fine but ideally you might want to run tomcat with a different user than tomcat8. In most installations Tomcat will mount a NFS directory with the data, in this case a better solution could be to have a common user (e.g. _opencga_) with the same id (uid) across the different servers.